pkg/operator/apis/monitoring/v1alpha1/types_integrations.go

```diff
@@ -0,0 +1,122 @@
+package v1alpha1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	apiextv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// IntegrationsSubsystemSpec defines global settings to apply across the
+// integrations subsystem.
+type IntegrationsSubsystemSpec struct {
+	// Label selector to find Integration resources to run. When nil, no
+	// integration resources will be defined.
+	Selector *metav1.LabelSelector `json:"selector,omitempty"`
+
+	// Label selector for namespaces to search when discovering integration
+	// resources. If nil, integration resources are only discovered in the
+	// namespace of the GrafanaAgent resource.
+	//
+	// Set to `{}` to search all namespaces.
+	NamespaceSelector *metav1.LabelSelector `json:"namespaceSelector,omitempty"`
```

Comment:
These are called `instanceSelector` and `instanceNamespaceSelector` for metrics and logs, so this is inconsistent with that. 

OTOH, those refer to selecting `MetricsInstances` and `LogsInstances`, so it might make sense to drop the prefix here.